assets/alignmentchart-style.css

```
@@ -1,23 +1,5 @@
-#alignment-body {
-    font-family: 'Open Sans', sans-serif;
-    padding: 5px;
-    background-color: #f2F7F7;
-    color: #505050;
-    height: calc(100vh - 65px);
-    overflow: auto;
-}
-
-#alignment-body ::-webkit-scrollbar {
-    width: 4px;
-    height: 0px;
-    margin-right: 500px;
-    background-color: #f2f7f7;
-}
 #alignment-body ::-webkit-scrollbar-thumb {
-    width: 2px;
-    background-color: #34ad99;
-    border-radius: 15px;
-    padding-right: 5px;
+    background-color: #34ad99 !important;
```

```suggestion
    background-color: #34AD99 !important;
```
(so it looks the same as line 25)